### DISCUSSION TITLE
parts (or new part-like structure?)

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#5378] [Ideas]
I opened this in the wrong place, https://github.com/ArctosDB/internal/discussions/218 - transferring.

----------

From observations committee discussion: https://docs.google.com/document/d/1hhbQHWXFvO2ax_jzrHLQwYUUAg6F0iH_18XBXqsppec/edit

Parts are currently only correctly one thing: physical stuff.

Parts are being used (incorrectly) for eg digital loans, by creating a media part to represent the online/digital object when there's no physical item.

Parts are being used (correctly) for tracking physical media, but it's not easy to separate eg "parts which might contain DNA" from "media."

Parts are being used (correctly, but not terribly directly) for representations (via the corresponding part attribute).

Parts are not being used to eg, describe the contents of (media/digital) images ("picture of a femur"), but there's interest in doing so.

Parts are not being used to eg, represent writing in field notes ("femur is ...."), but there's interest in doing so.

Etc.

The most-correct approach might be to introduce more tables beside parts (children of cataloged items).

An easier (reuse all forms, no new tables), more flexible (data-driven), and hopefully-correct-enough approach might be to add a new 'part category' (or something similar) column, values (and interpretation thereof) which might start as...

    physical (what we have now, expected to be bloody-bits in NH collections)
    media (what we do with 'media' part now - a way to track prints - but with an explicit 'not a bloody bit' flag, might make the possibility of lots of media-parts - "35mm color slide" - more approachable [or maybe that's still attributes??])
    representation (more-explicit replacement-or-repeat of the attribute, easier filtering/query but essentially no change. Used for eg case of bone; something physical but not something hacked off the individual represented by the catalog record)
    digital-or-whatever (functionally: pretend table coll_obj_cont_hist doesn't exist, no link to physical, this is something new, an expanded use of the table, not sure if this would be one value or one for image, one for audio, one for written, ....)

Is there some compelling reason to do this by adding tables?

Is there some compelling reason to NOT do this by adding some sort of not-null part categorization?

Would anyone actually use this if it were available?

Can/should we do this without https://github.com/ArctosDB/arctos/discussions/5231 (stable part identifiers)?

Can/should we do this without a media-->part linkage?

@ccicero @Jegelewicz @campmlc what'd I miss/muck up?

----

from @Jegelewicz 

Something I just encountered: https://docs.google.com/document/d/1t3oJqx4IitgPWeUg6EtylAmL2bXqZC1LVjtqAWaezqw/edit#heading=h.rbrfonilqbmg

An observation with a media part (for a digital image), but that “part” is more appropriate to an identification of [Digital Document](https://arctos.database.museum/name/Digital%20Document#secclass) or [Photograph](https://arctos.database.museum/name/Photograph#GettyAATviaArctos). Images without any biological information would be identified in one of these ways. I guess it just made it clear that we are doing things in many different ways. A cultural catalog item identified as a Photograph, uses the part name “object” and the catalog item type “item” - see https://arctos.database.museum/guid/ALMNH:EH:1102.

This does not clarify @dustymc proposal, but we are mixing part names, identification names and possibly media metadata for similar stuff.

-------

from @dustymc 

    more appropriate ... mixing

I don't think so.

There's a photograph - it's a part because we need to link it into the object tracking system. (And maybe some of the above will lead to other use cases.)

I can use the photograph to make an identification.

I can make an identification off of another photograph that's not linked to the record for all sorts of reasons (eg, I might use MVZ's high-res private copy).

I can create the photograph at an event.

I can make another ID based on the call I heard, but didn't capture, when I took the photo.

Etc etc etc - the same thing, or different things with similar properties, might be used in various places for various purposes, but no valid cross-node assumptions can be made from any of those things existing. There's plenty of opportunity to confound things, but I think that's easy enough to avoid as well.

----- 

@dustymc 

    l item identified as Photograph use the part media

I want to say "No, for the same reasons an item identified as a moose isn't limited to part=moose" or maybe "It's complicated, parts and identifications are fuzzy in cultural collections" but that fuzz might be evidence that I don't adequately understand SOMETHING, so ????

----- 

@campmlc 

But parts are physical things, and the photograph is a physical thing??? I think we should be consistent between a photo of a moose as a media part of the moose record and a photograph of a village in a cultural collection.

**Comment by AJLinn** [comment]
My first comment to this string of ideas is that at this point, I'm not going to stop the "indefensible" use of the media part to represent both the digital media scans of physical media that are useless for online collection users unless I digitize them, and the physical media (print, 35mm slide) that I could put in a protective sleeve and "stick a barcode" on. 

This particular usage of part: media and the subsequent attaching those parts to media loans, gives me, in an instant, the ability to see how many times an image a particular object is being used by the world. (See for example: https://arctos.database.museum/guid/UAM:EH:UA2003-013-0001
<img width="1403" alt="Screenshot 2023-07-05 at 12 16 13 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/f6b0c82e-807d-47aa-92c4-abaa07cb80c8">

There is no other field in Arctos that gives me the ability to conduct this major collection activity with such ease and sophistication. Within the media loan I can even document the transfer of the image file to my user as a "shipment"

<img width="1352" alt="Screenshot 2023-07-05 at 12 24 13 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/0448ba68-a281-4a34-ad11-5b1f2f0aa5f6">

And then add the citation once the publication is transferred to us.
I'd like to be included in ongoing discussions about part: media please.

**Reply by Jegelewicz** [reply]
@AJLinn we are working on this in Media Committee - next meeting is July 10 at 10 AM Pacific Time

**Reply by AJLinn** [reply]
I'll be there!

**Comment by dustymc** [comment]
20240108 Media meeting: would adding media_id FKEY-->media.media_id to table specimen_part accomplish the core of the issue?
